devicemaps.json

```diff
@@ -2029,167 +2029,18 @@
-              }
-            }
-          },
-          {
-            "pciAddress": "0000:81:00.2",
-            "type": "LAN",
-            "speed": 1000,
-            "name": "ge-0-2-2",
-            "description": "Port 5 labeled on the device",
-            "bcpNetwork": {
-              "standaloneBranch": {
-                "name": "lan5",
-                "description": "LAN 5 network interface, connected to port 5 labeled on the device"
-              }
-            }
-          },
-          {
-            "pciAddress": "0000:81:00.3",
-            "type": "LAN",
-            "speed": 1000,
-            "name": "ge-0-2-3",
-            "description": "Port 6 labeled on the device",
-            "bcpNetwork": {
-              "standaloneBranch": {
-                "name": "lan6",
-                "description": "LAN 6 network interface, connected to port 6 labeled on the device"
-              }
-            }
-          },
-          {
-            "pciAddress": "0000:c1:00.0",
-            "type": "LAN",
-            "speed": 10000,
-            "name": "xe-0-3-0",
-            "description": "Port 7 labeled on the device",
-            "bcpNetwork": {
-              "standaloneBranch": {
-                "name": "lan7",
-                "description": "LAN 7 network interface, connected to port 7 labeled on the device"
-              }
-            }
-          },
-          {
-            "pciAddress": "0000:c1:00.1",
-            "type": "LAN",
-            "speed": 10000,
-            "name": "xe-0-3-1",
-            "description": "Port 8 labeled on the device",
-            "bcpNetwork": {
-              "standaloneBranch": {
-                "name": "lan8",
-                "description": "LAN 8 network interface, connected to port 8 labeled on the device"
-              }
-            }
-          },
-          {
-            "pciAddress": "0000:c1:00.2",
-            "type": "LAN",
-            "speed": 10000,
-            "name": "xe-0-3-2",
-            "description": "Port 9 labeled on the device",
-            "bcpNetwork": {
-              "standaloneBranch": {
-                "name": "lan9",
-                "description": "LAN 9 network interface, connected to port 9 labeled on the device"
-              }
-            }
-          },
-          {
-            "pciAddress": "0000:c1:00.3",
-            "type": "LAN",
-            "speed": 10000,
-            "name": "xe-0-3-3",
-            "description": "Port 10 labeled on the device",
-            "bcpNetwork": {
-              "standaloneBranch": {
-                "name": "lan10",
-                "description": "LAN 10 network interface, connected to port 10 labeled on the device"
+                "description": "HA fabric network interface, connected to port 2/3 labeled on the device"
               }
             }
           }
         ],
         "lte": []
       }
+      }
```

Comment:
My validation is failing on
```
E           json.decoder.JSONDecodeError: Extra data: line 2046 column 1 (char 65615)
```
I believe because of this addition. Please remove